Jenkinsfile.release

```
@@ -94,7 +94,7 @@ pipeline {
                 script {
                     echo "Call ${CREATE_RELEASE_BRANCHES} job"
 
-                    def buildParams = [:]
+                    def buildParams = getDefaultBuildParams()
```

Oh I see now ...
Correction should just have been this one. No need to use `getDefaultBuildParams` and put all other parameters in the call.

```suggestion
                    def buildParams = []
```

@cristianonicolai Could be set like that after the release IMO ;)